standalone/inc/pup/PUPManager.cpp

```diff
@@ -230,14 +230,10 @@ TTF_Font* PUPManager::GetFont(const string& szFont)
 
 void PUPManager::QueueTriggerData(PUPTriggerData data)
 {
-   if (data.value == 0)
-      return;
-
    {
```

Comment:
I remember specifically adding this. I feel like it was related to dof or something, but this would get hammered.

Reply:
Pup triggers can trigger when a value goes to 0. Not sure why dof is related as this is in the pup code?

Reply:
As I said, it was a long time since I wrote that, but I do remember specifically putting that in for a reason.  We'll see what happens.